src/main/java/github/weichware10/analyse/gui/TrialSelector.java

```diff
@@ -0,0 +1,32 @@
+package github.weichware10.analyse.gui;
+
+import github.weichware10.analyse.gui.util.AbsScene;
+import java.util.Optional;
+
+import javafx.scene.control.ButtonType;
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.imports.CustomImportOrderCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Extra separation in import group before 'javafx.scene.control.ButtonType'